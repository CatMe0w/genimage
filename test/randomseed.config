image randomseed.img {
    hdimage {
        partition-table-type = "gpt"
    }

    partition rootfs {
        size = 1M
    }
}

config {
    randomseed = 1
}
