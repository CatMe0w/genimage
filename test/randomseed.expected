00000000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
000001c0  02 00 ee 21 29 00 01 00  00 00 47 08 00 00 00 00  |...!).....G.....|
000001d0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
000001f0  00 00 00 00 00 00 00 00  00 00 00 00 00 00 55 aa  |..............U.|
00000200  45 46 49 20 50 41 52 54  00 00 01 00 5c 00 00 00  |EFI PART....\...|
00000210  a8 9f e4 6e 00 00 00 00  01 00 00 00 00 00 00 00  |...n............|
00000220  47 08 00 00 00 00 00 00  22 00 00 00 00 00 00 00  |G.......".......|
00000230  26 08 00 00 00 00 00 00  18 07 21 30 00 55 81 53  |&.........!0.U.S|
00000240  ff 99 08 fd 68 07 00 40  02 00 00 00 00 00 00 00  |....h..@........|
00000250  80 00 00 00 80 00 00 00  e5 81 84 df 00 00 00 00  |................|
00000260  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00000400  af 3d c6 0f 83 84 72 47  8e 79 3d 69 d8 47 7d e4  |.=....rG.y=i.G}.|
00000410  62 09 50 60 00 40 99 61  ff 94 03 fd 88 82 00 46  |b.P`.@.a.......F|
00000420  22 00 00 00 00 00 00 00  21 08 00 00 00 00 00 00  |".......!.......|
00000430  00 00 00 00 00 00 00 00  72 00 6f 00 6f 00 74 00  |........r.o.o.t.|
00000440  66 00 73 00 00 00 00 00  00 00 00 00 00 00 00 00  |f.s.............|
00000450  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00104e00  af 3d c6 0f 83 84 72 47  8e 79 3d 69 d8 47 7d e4  |.=....rG.y=i.G}.|
00104e10  62 09 50 60 00 40 99 61  ff 94 03 fd 88 82 00 46  |b.P`.@.a.......F|
00104e20  22 00 00 00 00 00 00 00  21 08 00 00 00 00 00 00  |".......!.......|
00104e30  00 00 00 00 00 00 00 00  72 00 6f 00 6f 00 74 00  |........r.o.o.t.|
00104e40  66 00 73 00 00 00 00 00  00 00 00 00 00 00 00 00  |f.s.............|
00104e50  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00108e00  45 46 49 20 50 41 52 54  00 00 01 00 5c 00 00 00  |EFI PART....\...|
00108e10  e3 54 c1 02 00 00 00 00  47 08 00 00 00 00 00 00  |.T......G.......|
00108e20  01 00 00 00 00 00 00 00  22 00 00 00 00 00 00 00  |........".......|
00108e30  26 08 00 00 00 00 00 00  18 07 21 30 00 55 81 53  |&.........!0.U.S|
00108e40  ff 99 08 fd 68 07 00 40  27 08 00 00 00 00 00 00  |....h..@'.......|
00108e50  80 00 00 00 80 00 00 00  e5 81 84 df 00 00 00 00  |................|
00108e60  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
*
00109000
